src/wp-includes/cache.php

```diff
@@ -11,6 +11,8 @@
 /** WP_Object_Cache class */
 require_once ABSPATH . WPINC . '/class-wp-object-cache.php';
 
+define( 'WP_OBJECT_CACHE_SUPPORTS_GROUP_FLUSH', false );
```

Comment:
The default in-memory cache support group flushes.

```suggestion
/**
  * Indicate that this object cache implementation supports group flushing.
  */
define( 'WP_OBJECT_CACHE_SUPPORTS_GROUP_FLUSH', true );
```